packages/react-aria-components/src/ListBox.tsx

```diff
@@ -120,7 +120,7 @@ interface ListBoxInnerProps<T> {
 }
 
 function ListBoxInner<T extends object>({state: inputState, props, listBoxRef}: ListBoxInnerProps<T>) {
-  let {filter, collectionProps, collectionRef} = useContext(UNSTABLE_InternalAutocompleteContext) || {};
+  let {filter, collectionRef, ...collectionProps} = useContext(CollectionContext) || {};
```

Comment:
are we still using `UNSTABLE_useFilteredListState` as the name for this? or should these also be updated out of UNSTABLE?

not actually attached to this line, but it's nearby